docs/content/users/quickstart.md

```diff
@@ -70,6 +70,45 @@ Please note that you will need to change the PHP version to 7.4 to be able to wo
     ddev launch
     ```
 
+## Contao
+
+Further information can also be found in the [Contao documentation](https://docs.contao.org/manual/en/guides/local-installation/ddev/).
+The [Contao demo website](https://demo.contao.org/), with demonstrations and examples of some of the features this CMS, can be added when installing the CMS and is designed to help you get started with your first web projects.
+The demo website can be optionally installed alongside Contao, either through the [Contao Manager or with Composer](https://github.com/contao/contao-demo). 
+
+=== "Composer"
+
+    ```bash
+    mkdir -p ~/projects/my-contao && cd ~/projects/my-contao
+    ddev config --project-type=php --docroot=public --webserver-type=apache-fpm --php-version=8.2 --create-docroot --timezone=Europe/Berlin
+    ddev composer create contao/managed-edition:5.3
+    
+    # After installation, the database access data must be entered in the .env.local file. At the same time, we also set up Mailpit here too:
+    # DATABASE_URL=mysql://db:db@db:3306/db
+    # MAILER_DSN=smtp://localhost:1025
+    
+    # Create the database
+    ddev exec "bin/console contao:migrate"
+    
+    # Create backend user
+    ddev exec "bin/console contao:user:create"
+    
+    # The administration area can be accessed directly: ddev launch /contao
+    ddev launch
+    ```
+
+=== "Contao Manager"    
+
+    Like most PHP projects, Contao could be installed and updated with Composer. The [Contao Manager](https://docs.contao.org/manual/en/installation/contao-manager/) is a tool that provides a graphical user interface to manage a Contao installation.
+
+    ```bash
+    mkdir -p ~/projects/my-contao && cd ~/projects/my-contao
+    ddev config --project-type=php --docroot=public --webserver-type=apache-fpm --php-version=8.2 --create-docroot --timezone=Europe/Berlin
+    ```
+
+    Afterwards you can [download the Contao Manager](https://contao.org/en/download), copy it into the public directory and rename it (.php). Start DDEV and follow the further steps within the Contao Manager.
+
```

Comment:
````suggestion
    mkdir my-contao-site && cd my-contao-site
    ddev config --project-type=php --docroot=public --webserver-type=apache-fpm
    ddev start

    # Download the Contao Manager
    ddev exec "wget -O public/contao-manager.phar.php https://download.contao.org/contao-manager/stable/contao-manager.phar"

    # Follow the further steps within the Contao Manager, insert mysql://db:db@db:3306/db when asked for database
    ddev launch /contao-manager.phar.php
    ```
````

Reply:
Thank you very much. I have included the corresponding updates in the last commit & moved the note on installing the Contao demo website